scripts/openstack-quickstart-demosetup

```diff
@@ -624,8 +624,13 @@ if ! rabbitmqctl list_users|grep -q openstack; then
 fi
 
 # Start Keystone and Neutron
+# TODO: Trying to enable both "neutron" and "neutron-server" here since 
+#       rpm-packaging is only providing "openstack-neutron" as an Alias and
+#       enabling by alias is not supported by systemctl. We can remove
+#       "openstack-neutron" here as soon as we switched to the rpm-packaging
+#       provided packages.
```

Comment:
According to https://www.freedesktop.org/software/systemd/man/systemd.unit.html

```
Alias names may be used in commands like enable, disable, start, stop, status, and similar, and in all unit dependency directives, including Wants=, Requires=, Before=, After=. Aliases cannot be used with the preset command.
```

Are you sure the alias is not working?

Reply:
> Are you sure the alias is not working?

Yes. At least for "Aliases" defined in the [Install] section (the symlinks for those are only created when enabling a unit).
The above snippet is about creating aliases by creating the symlinks manually. That's the part I forgot about. So I guess we can just create an "openstack-neutron.service" symlink in the upstream package.